<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>958</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>700</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(144, 0, 0);
border-color: rgb(122, 10, 10);
border-style: solid;
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>700</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>620</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>461</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ac">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>381</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(195, 195, 195);
border-style: solid;</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_left">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>460</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>540</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>620</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>700</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>620</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>380</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>460</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>540</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>620</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>460</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>540</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>620</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>700</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>380</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>380</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>460</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>540</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>620</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_credit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>700</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>460</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>540</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caladea</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(212, 212, 212);</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>681</width>
      <height>651</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>971</width>
      <height>941</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(view/pic.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>621</width>
      <height>131</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-style: solid;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>621</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_text">
        <property name="font">
         <font>
          <family>Caladea</family>
          <pointsize>24</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_ac</zorder>
   <zorder>pushButton_0</zorder>
   <zorder>pushButton_minus</zorder>
   <zorder>pushButton_mult</zorder>
   <zorder>pushButton_acos</zorder>
   <zorder>pushButton_tan</zorder>
   <zorder>pushButton_left</zorder>
   <zorder>pushButton_asin</zorder>
   <zorder>pushButton_equal</zorder>
   <zorder>pushButton_log</zorder>
   <zorder>pushButton_credit</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_pow</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_graph</zorder>
   <zorder>pushButton_sin</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_atan</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_dot</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_mod</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_ln</zorder>
   <zorder>pushButton_cos</zorder>
   <zorder>pushButton_div</zorder>
   <zorder>pushButton_plus</zorder>
   <zorder>pushButton_right</zorder>
   <zorder>pushButton_sqrt</zorder>
   <zorder>scrollArea</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
